release/serve_tests/workloads/microbenchmarks.py

```diff
@@ -127,21 +127,25 @@ async def _main(
     # HTTP
     if run_http:
         if run_latency:
-            serve.run(Noop.bind())
             # Microbenchmark: HTTP noop latencies
+            serve.run(Noop.bind())
```

Comment:
A bit nitpick, but we can probably DRY those up to have them running in a loop. 

Also I think it might also be useful we do a serve shutdown in between each run to have all the component start over again.

Reply:
sounds good, put latency benchmarks in a loop and added serve shutdown in between